aws-amplify/docs
# a11y videos have aria-hidden but can still receive focus

#### Description of changes:
[Issue] Video controls can be focused with the keyboard, but they are hidden from  assistive technologies using aria-hidden="true".
[Fix] remove `aria-hidden="true"` and add aria-label providing description of video.

To test manually:
- Turn on screen reader (for Mac, System Settings > Accessibility > VoiceOver)
- Go to staging site: https://a11y-video-focus.d1egzztxsxq9xz.amplifyapp.com/react/deploy-and-host/fullstack-branching/custom-pipelines/#set-up-backend-deployments
- Keyboard navigate to a video
- Tab through the controls in the video to ensure that video controls are announced.

Example of build failure when `ariaDescription` prop is missing from a `<Video />` element: 

![Screenshot 2024-07-31 at 11 58 17 AM](https://github.com/user-attachments/assets/a1c614fa-196e-4a0c-a0e7-c6daaa504cfd)

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
